monitoring_dashboard.qmd

```diff
@@ -0,0 +1,138 @@
+---
+title: "Monitoring dashboard"
```

Comment:
Is this file in two places, in the root of the repo as well as in `templates/`? Is that right?